crates/topos/tests/snapshots/regtest__regtest_spam_help_display.snap

```diff
@@ -33,6 +33,12 @@ Options:
           Socket of the opentelemetry agent endpoint. If not provided open telemetry will not be used [env: TOPOS_OTLP_AGENT=]
       --otlp-service-name <OTLP_SERVICE_NAME>
           Otlp service name. If not provided open telemetry will not be used [env: TOPOS_OTLP_SERVICE_NAME=]
+      --benchmark
+          Flag to indicate usage of Kubernetes [env: TOPOS_NETWORK_SPAMMER_BENCHMARK=]
+      --dns <DNS>
+          DNS template for generating target node addresses. e.g. `--dns="http://validator-{N}:1340"` [env: TOPOS_NETWORK_SPAMMER_DNS=]
+      --number <NUMBER>
+          Number of nodes to generate based on the DNS template [env: TOPOS_NETWORK_SPAMMER_NUMBER=]
```

Comment:
So using `--number` without both `--dns` and `--benchmark` would be an error? I'm a bit concerned about having interdependent switches like that. Can get messy.

Reply:
`clap` can `require` flags if another flag is passed. However, we are using `#[derive]` and it doesn't work (yet?) there. I don't know how else to solve this but just by adding maybe a help text?